SwiftTweaks/TweakTableCell.swift

```diff
@@ -208,7 +208,7 @@ internal final class TweakTableCell: UITableViewCell {
 			let doubleValue = viewData.doubleValue!
 			self.updateStepper(value: doubleValue, stepperValues: viewData.stepperValues!)
 
-			textField.text = String(doubleValue)
+			textField.text = String(describing: viewData.value)
```

Comment:
I don't know if it's idiomatic or not, if possible I would propose to implement `CustomStringConvertible` for `TweakViewData` so that each of the type have their own UI string representation (I will update the PR if you guys ok with that), for now I'll use this approach.